### PR TITLE
Use mocking instead of file system which requires sudo

### PR DESCRIPTION
**Pull request by SamMousa**
| Q             | A
| ------------- | ---
| Is bugfix?    | n/a
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #2

### CODE REVIEW

**Comment by cebe** [comment]
this could probably be dropped from travis.yml: https://github.com/yiisoft/cache/blob/95dba0763b1376dfd07fa1f25ac98fc114a09815/.travis.yml#L34

**Comment by SamMousa** [comment]
There's more cleanup possible, i'll get into it.

File: tests/unit/DbDependencyTest.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 /**
  * @group caching
+ * @skp
```
Comment (samdark) [inline tests/unit/DbDependencyTest.php]: skp?

**Comment by samdark** [comment]
Solved by https://github.com/yiisoft/cache/pull/16 Thanks for ideas. It is much faster with tests now. Going to apply similar settings to other repositories later.

**Pull request closed without merging**